app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/Router.scala

```diff
@@ -140,11 +144,12 @@ trait JourneyTypeRoutes extends Product with Serializable {
   //TODO: fix routing to go to ALF address lookup page for enter contact details
   // if coming from CheckClaimantDetails from the change link => next page is claimRoutes.CheckClaimantDetailsController.show(journeyBindable)
   // if coming from CheckClaimantDetails from selecting 'yes' in radio buttons (i.e no data from Acc14) => next page is ALF address page
```

Comment:
What this comment is about? I don't really understand it? 
When I should call ALF? I was thinking then user clicks on change button, isn't?